newIDE/app/src/ProjectsStorage/ResourceMover/LocalResourceMover.js

```diff
@@ -48,6 +227,24 @@ const movers: {
       fileMetadata: newFileMetadata,
       onProgress,
     }),
+
+  // Moving to GDevelop "Cloud" storage:
+
+  //TODO
```

Comment:
todo?

Reply:
This was done already - I updated the comment :)